src/neptune_sklearn/impl/__init__.py

```diff
@@ -217,17 +226,26 @@ def create_classifier_summary(classifier, X_train, X_test, y_train, y_test, nrow
         "scores": get_scores(classifier, X_test, y_test, y_pred=y_pred),
     }
 
-    if log_charts:
-        cls_summary["diagnostics_charts"] = {
-            "classification_report": create_classification_report_chart(classifier, X_train, X_test, y_train, y_test),
-            "confusion_matrix": create_confusion_matrix_chart(classifier, X_train, X_test, y_train, y_test),
-            "ROC_AUC": create_roc_auc_chart(classifier, X_train, X_test, y_train, y_test),
-            "precision_recall": create_precision_recall_chart(classifier, X_test, y_test),
-            "class_prediction_error": create_class_prediction_error_chart(classifier, X_train, X_test, y_train, y_test),
-        }
-
     cls_summary["integration/about/neptune-sklearn"] = __version__
 
+    if log_charts:
+        classification_report = create_classification_report_chart(classifier, X_train, X_test, y_train, y_test)
+        confusion_matrix = create_confusion_matrix_chart(classifier, X_train, X_test, y_train, y_test)
+        ROC_AUC = create_roc_auc_chart(classifier, X_train, X_test, y_train, y_test)
```

Comment:
how about renaming `ROC_AUC` to lower case. Upper case in python is usually reserved for global const vars

Reply:
We could, but it would be a breaking change for users trying to access the `ROC_AUC` programmatically.